File: zig/b3.zig
const std = @import("std");
pub export fn _start() callconv(.Naked) noreturn {
    std.os.exit(modify(76));
}
fn modify(n: u8) u8 {
    const buffer = [1]u8 { @intCast(u8, 0xFF & n) };
    return @intCast(u8, 0xFF & djb2(&buffer));
}
fn djb2(msg: []const u8) usize {
    var hash : u32 = 5381;
    var i : usize = 0;
    while (i < msg.len) : (i += 1) {
        hash = ((hash << 5) + hash) + msg[i];
    }
    return hash;
}

File: c/b3.c
#include "exit_syscall.h"
static int djb2(const unsigned char *msg, unsigned len)
{
  int hash = 5381;
  for (unsigned i = 0; i < len; i++) {
    hash = ((hash << 5) + hash) + msg[i];
  }
  return hash;
}
static int modify(int n)
{
  unsigned char msg[2] = {(unsigned char)(0xFF & n)};
  return djb2(msg, 1);
}
__attribute__((noreturn))
void _start()
{
  exit(modify(76));
  __builtin_unreachable();
}
